wct-test/json-forms.html

```diff
@@ -0,0 +1,31 @@
+<!doctype html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <script src="../node_modules/webcomponents.js/webcomponents-lite.js"></script>
+    <script src="../node_modules/web-component-tester/browser.js"></script>
+    <script src="./native-shim.js"></script>
+    <script src="../dist/jsonforms.js"></script>
+</head>
+<body>
+<script>
+    suite('json-forms element', function() {
+        test('is awesomest', function() {
```

Comment:
Yes, although we should be able to get some coverage via [this](https://github.com/thedeeno/web-component-tester-istanbul) or something similar.